infomaniak-build-tools/macos/build-drive.sh

```diff
@@ -97,7 +97,7 @@ cmake \
 make
 
 # Generate Debug files
-dsymutil ./bin/kDrive.app/Contents/MacOS/kDrive -o ../kDrive.dSYM
+dsymutil ./install/kDrive.app/Contents/MacOS/kDrive -o ../kDrive.dSYM
```

Comment:
This change is related to the CMakeLists change for MacOS, building the kDrive server app in the install directory instead of bin